docs/src/components/sender.md

```
@@ -79,273 +175,616 @@ Sender 支持单行和多行两种输入模式，通过 `mode` 属性控制。
+| maxLength | 最大输入长度 | `number` | `Infinity` |
+| showWordLimit | 是否显示字数统计 | `boolean` | `false` |
+| submitType | 提交方式 | `'enter' \| 'ctrlEnter' \| 'shiftEnter'` | `'enter'` |
+| stopText | 停止按钮文字 | `string` | `'停止响应'` |
+| defaultActions @0.4.0 | 默认操作按钮配置 | `DefaultActions` | `undefined` |
+| extensions @0.4.0 | 扩展列表 (Template, Mention, Suggestion 等) | `Extension[]` | `[]` |
+
+:::tip 扩展系统
+使用 `extensions` 属性配置功能扩展，提供灵活的配置和完整的类型支持。
+::: 
+
+#### Template
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)